src/domain/event/__tests__/CreateEventPage.test.tsx

```diff
@@ -40,7 +41,7 @@ import CreateEventPage from '../CreateEventPage';
 expect.extend(toHaveNoViolations);
 
 beforeEach(() => {
-  jest.spyOn(Router, 'useHistory').mockReturnValue({});
+  jest.spyOn(Router, 'useHistory').mockReturnValue({} as any);
```

Comment:
Are these mocks needed? Maybe you could wrap component with MemoryRouter by using render function from testUtils

Reply:
Nope not needed. I just didn't refactor these yet